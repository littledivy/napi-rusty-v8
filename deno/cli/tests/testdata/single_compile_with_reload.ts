await import("./single_compile_with_reload_dyn.ts");
console.log("1");
await import("./single_compile_with_reload_dyn.ts");
console.log("2");
await new Promise((r) =>
  new Worker(
    new URL("single_compile_with_reload_worker.ts", import.meta.url).href,
    { type: "module" },
  ).onmessage = r
);
console.log("3");
await new Promise((r) =>
  new Worker(
    new URL("single_compile_with_reload_worker.ts", import.meta.url).href,
    { type: "module" },
  ).onmessage = r
);
console.log("4");
