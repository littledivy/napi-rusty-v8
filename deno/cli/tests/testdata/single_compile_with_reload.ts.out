Check [WILDCARD]single_compile_with_reload.ts
Hello
1
2
Check [WILDCARD]single_compile_with_reload_worker.ts
Hello from worker
3
Hello from worker
4
